app/View/Layouts/default.ctp

```diff
@@ -57,16 +57,17 @@
     </div>
 </nav>
 <!-- Player -->
-<div class="navbar navbar-default navbar-fixed-top navbar-player">
+<div class="navbar navbar-default navbar-fixed-bottom navbar-player">
     <div class="container">
-        <!-- Play/Pause buttons -->
-        <div class="col-md-2 col-xs-3">
+        <!-- Playlist management -->
+        <div class="col-md-2 col-xs-1">
             <ul class="player-controls">
```

Comment:
Since backward, play and forward are replaced by completely different options (queue-button, queue-repeat and queue-shuffle), is this class still `player-controls`?

Reply:
Yes, at first the playlist management section already had a player-controls class.
I've switched the two sections because most people are right handed and on mobile view it was hiding the artist and song title when skipping song.